src/app.d.ts

```diff
@@ -2,7 +2,10 @@
 // for information about these interfaces
 declare global {
   namespace App {
-    // interface Error {}
+    interface Error {
+      // `message` is included by default
+      title?: string;
```

Comment:
We'll default to the error code if a title isn't provided, but I figured it would be good to start stubbing out other fields here we could use for error messaging.